engine/CMakeLists.txt

```diff
@@ -842,21 +842,26 @@ IF (WIN32 AND NOT UNIX)
     SET(CPACK_NSIS_DISPLAY_NAME "Vega Strike")
     SET(CPACK_NSIS_HELP_LINK "https:\\\\vega-strike.org")
     SET(CPACK_NSIS_URL_INFO_ABOUT "https:\\\\vega-strike.org/about")
-    SET(CPACK_NSIS_CONTACT "TBD@vega-strike.org")
+    SET(CPACK_NSIS_CONTACT "maintainers@vega-strike.org")
     SET(CPACK_NSIS_MODIFY_PATH ON)
 
-    # NSIS, Wix, and compressed archives (7z, Zip)
-    SET(CPACK_GENERATOR "NSIS" "NSIS64" "WIX" "7Z" "ZIP")
-    SET(CPACK_PACKAGE_EXECUTABLES "vegastrike.exe" "vegastrike-engine.exe" "vega-meshtool.exe" "vegasettings.exe")
-ELSEIF (XCODE)
-    MESSAGE(STATUS "Configuring Packaging for Mac OS X")
+    # Windows installer format(s) to output
+    SET(CPACK_GENERATOR "NSIS64")
```

Comment:
Just to note - https://cmake.org/cmake/help/latest/module/CPack.html#variable:CPACK_GENERATOR
it's basically a list of generators that could be supported. I'm okay with us dropping it down to just one generator - NSIS64.